game/forms.py

```
@@ -1,21 +1,56 @@
+    email = forms.EmailField(
+        required=True,
+        widget=forms.EmailInput(attrs={
+            'autocomplete': 'email'
+        })
+    )
+
 class CustomUserCreationForm(UserCreationForm):
     email = forms.EmailField(required=True)
 
     class Meta(UserCreationForm.Meta):
         fields = UserCreationForm.Meta.fields + ('email',)
```

Ruff (0.15.13)

[warning] Consider `(*UserCreationForm.Meta.fields, 'email')` instead of concatenation

Replace with `(*UserCreationForm.Meta.fields, 'email')`

(RUF005)